guides/source/5_0_release_notes.md

```diff
@@ -227,6 +227,9 @@ Please refer to the [Changelog][action-pack] for detailed changes.
     `RedirectBackError`.
     ([Pull Request](https://github.com/rails/rails/pull/22506))
 
+*   `ActionDispatch::IntegrationTest` and `ActionController::TestCase` deprecate positional arguments in favor of
+    positional arguments. ([Pull Request](https://github.com/rails/rails/pull/18323))
```

Comment:
I'm glad we deprecated positional arguments in favor of positional arguments. Positional arguments are much nicer to use than positional arguments. :trollface: